tests/test_simulation.py

```diff
@@ -80,16 +81,18 @@ def test_no_network_config():
         with edit_config(config_path) as config:
             config.pop("network")
 
+        os.remove(config_path.parent / "circuit_config.json")
         simulation = test_module.Simulation(config_path)
 
-        with pytest.raises(BluepySnapError, match="No 'network' set"):
+        with pytest.raises(BluepySnapError, match="'network' file not found"):
```

Comment:
The test is now different and should be renamed from `test_no_network_config` to something like `test_network_file_not_found`.
However, maybe the old test `test_no_network_config` could be kept to verify also that case.

Reply:
I'll rename the test if need be. We should discuss and decide on the libsonata behavior first, though.

Reply:
Yeah, no need to add `test_no_network_config` as I was thinking in the previous comment. I would just rename the name of the test.

Reply:
> renamed